.eslintrc

```diff
@@ -2,8 +2,12 @@
   "parser": "babel-eslint",
   "extends": "airbnb/base",
   "rules": {
+    "no-underscore-dangle": 0,
+    "no-duplicate-imports": 0,
+    "global-require": 0,
     "semi": [2, "never"],
     "prefer-arrow-callback": 0,
+    "import/no-unresolved": [2, { ignore: ["atom"] }],
```

Comment:
`npm` (and thus _any_ future Greenkeeper PR) will expand this out to multiple lines, run a `npm --save <package that is already listed>` to see what it will do.